plugins/optimization-detective/site-health.php

```diff
@@ -156,7 +157,14 @@ function od_compose_site_health_result( $response ): array {
 				$result['description'] .= '<blockquote>' . esc_html( $data['message'] ) . '</blockquote>';
 			}
 
-			$result['description'] .= '<details><summary>' . esc_html__( 'Raw response:', 'optimization-detective' ) . '</summary><pre style="white-space: pre-wrap">' . esc_html( $body ) . '</pre></details>';
+			$result['description'] .= '<details><summary>' . esc_html__( 'Raw response:', 'optimization-detective' ) . '</summary>';
+
+			if ( isset( $header ) && 'text/html' === $header ) {
```

Comment:
This will account for headers values like `text/html; charset=utf-8` and `application/xml+xhtml`:

```suggestion
			if ( is_string( $header ) && str_contains( $header, 'html' ) {
```

Reply:
Aside: `$header` will always be set and never be `null` so the use of `isset()` probably was unnecessary, right?